### PR TITLE
feature: prepare production achievements

### PR DESCRIPTION
**Pull request by dhenkel92**
The idea is that we can copy & execute the `graphql-seed` & `graphql-enable` mutations in our production apollo playground.

The alternative would be writing a script to execute the queries, but I don't see an advantage.

### CODE REVIEW

File: common/achievement/template.ts
Comment (dhenkel92) [inline common/achievement/template.ts:171]: Subtitle is not required

File: common/achievement/template.ts
Comment (dhenkel92) [inline common/achievement/template.ts:175]: There might be actions that don't have a link, like waiting for course approval.

File: common/achievement/template.ts
Comment (dhenkel92) [inline common/achievement/template.ts:195]: This was failing if valueToAchieve is 0

File: graphql/achievement_template/mutations.ts
Comment (dhenkel92) [inline graphql/achievement_template/mutations.ts:42]: I wasn't able to generate the graphql json scalar via node. Therefore, I switched the input type to string and move the parsing to the resolver function.

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**
